go.mod

```diff
@@ -8,12 +8,12 @@ require (
 	github.com/giantswarm/apiextensions-application v0.3.1
 	github.com/giantswarm/app/v6 v6.8.1
 	github.com/giantswarm/appcatalog v0.6.0
-	github.com/giantswarm/apptest v1.1.1
+	github.com/giantswarm/apptest v1.2.0
```

Comment:
We need this fix to the apptest library.

I bumped the others too since renovate isn't working.